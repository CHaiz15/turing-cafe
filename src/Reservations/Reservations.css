.res-container {
  margin: 5% 0 0 0;
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
}