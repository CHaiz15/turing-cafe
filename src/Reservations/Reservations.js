import React from 'react';
import ReservationCard from '../ReservationCard/ReservationCard'
import './Reservations.css'


const Reservations = ({ reservations }) => {
  let allReservations = reservations.map(reservation => {
    return <ReservationCard id={reservation.id} name={reservation.name} date={reservation.date} time={reservation.time} number={reservation.number} />
  })
  return (
    <div className='res-container'>
      {allReservations}
    </div>
  )
}

export default Reservations