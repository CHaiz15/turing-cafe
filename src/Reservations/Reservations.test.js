import React from 'react';
import Reservations from './Reservations';
import { shallow } from 'enzyme'

it('should be an instance of a Reservations componenet', () => {
  let wrapper = shallow(
    <Reservations 
      reservations={[]}
    />);
  expect(wrapper).toMatchSnapshot();
})