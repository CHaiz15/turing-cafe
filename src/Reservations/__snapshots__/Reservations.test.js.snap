// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be an instance of a Reservations componenet 1`] = `
<div
  className="res-container"
/>
`;
